.github/workflows/unit-tests.yaml

```diff
@@ -24,4 +24,4 @@ jobs:
       run: make bin
 
     - name: Test
-      run: make test
+      run: sudo make test
```

Comment:
just to prevent us removing it in future, can you include a small comment about why we're using `sudo`?

specifically, we want to run as root for systemd tests https://github.com/awslabs/mountpoint-s3-csi-driver/blob/5cd426dc6c884020d445e7c2f9c1f94de8a92528/pkg/system/systemd_test.go#L180-L182